packages/fluentui/react-northstar-emotion-renderer/src/createEmotionRenderer.ts

```diff
@@ -0,0 +1,121 @@
+import createCache from '@emotion/cache';
+import { ObjectInterpolation, serializeStyles } from '@emotion/serialize';
+import { StyleSheet } from '@emotion/sheet';
+import { EmotionCache, insertStyles } from '@emotion/utils';
+import {
+  Renderer,
+  RendererRenderGlobal,
+  RendererRenderFont,
+  RendererRenderRule,
+} from '@fluentui/react-northstar-styles-renderer';
+// @ts-ignore No typings :(
+import focusVisiblePlugin from '@quid/stylis-plugin-focus-visible';
+// @ts-ignore No typings :(
+import rtlPlugin from 'stylis-plugin-rtl';
+import * as React from 'react';
+
+import { disableAnimations } from './disableAnimations';
+import { generateFontSource, getFontLocals, toCSSString } from './fontUtils';
+import { invokeKeyframes } from './invokeKeyframes';
+
+export function createEmotionRenderer(target?: Document): Renderer {
+  const cacheLtr = createCache({
+    container: target?.head,
+    key: 'fui',
+    stylisPlugins: [focusVisiblePlugin],
+
+    // TODO: make this configurable via perf flags
+    speedy: true,
+  }) as EmotionCache & { insert: Function };
+  const cacheRtl = createCache({
+    container: target?.head,
+    key: 'rfui',
+    stylisPlugins: [focusVisiblePlugin, rtlPlugin],
+
+    // TODO: make this configurable via perf flags
+    speedy: true,
+  });
+
+  const sheet = new StyleSheet({
+    key: `${cacheLtr.key}-global`,
+    nonce: cacheLtr.sheet.nonce,
+    container: cacheLtr.sheet.container,
+  });
+
+  const Provider: React.FC = props => {
+    // TODO: Find a way to cleanup global styles
+    // React.useEffect(() => {
+    // return () => sheet.flush();
+    // });
+
+    return React.createElement(React.Fragment, null, props.children);
```

Comment:
nit: why not JSX?

Reply:
I was too lazy to rename file to a `.tsx` 😺